packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/advanced-duration.tsx

```diff
@@ -49,7 +87,7 @@ const AdvancedDuration = observer(
             'has-time': is_24_hours_contract,
         });
 
-        const changeExpiry = ({ target }) => {
+        const changeExpiry = ({ target }: { target: { name: string; value: string | number | boolean } }) => {
```

Comment:
Here the value might have so may types because of `onChangeUiStore({ name, value });` on line 94. 
`onChangeUiStore` is used for changing to many different observables, like:
`onChangeUiStore({ name: 'is_advanced_duration', value: true })`; **boolean** in _duration-wrapper.tsx_
`onChangeUiStore({ name: 'advanced_expiry_type', value: 'endtime' });` **string** in _duration-wrapper.tsx_
`onChangeUiStore({ name: `duration_${duration_unit}`, value: min_value });` **number** _in mobile-widget_

Reply:
@kate-deriv in this case we need to use `unknown` for `value` here. :)

Reply:
Will do, thank you